src/pages/buy-sell/screens/BuySellTable/BuySellTable.tsx

```diff
@@ -82,7 +91,7 @@ const BuySellTable = () => {
                     list={getSortByList(localize)}
                     onRequestClose={hideModal}
                     onToggle={onToggle}
-                    selected={sortDropdownValue as string}
+                    selected={sortByValue as string}
```

Comment:
```suggestion
                    selected={sortByValue}
```
can we remove string coercions?